multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -588,8 +592,14 @@ def _hyperparameter_tune(self, hyperparameter_tune_kwargs, resources, **_fit_arg
         if _fit_args.get("teacher_predictor", None) is not None:
             is_distill = True
         try:
+            run_config_kwargs = {
+                "checkpoint_config": CheckpointConfig(
+                    num_to_keep=3,
```

Comment:
wondering any reason why we want to keep track of the best 3?

Reply:
Just a random number we decided to put for now.